app/views/kudo_mailer/batch_kudo_notification.html.erb

```diff
@@ -1,10 +1,10 @@
 <% content_for :message do %>
 
   <% @commentables.each_with_index do |commentable, index| %>
-  
-    <% givers_list = @kudo_givers["#{commentable.class.name}_#{commentable.id}"] %>
-    <%= givers_list.each_with_index.map {|g,i| style_bold((g == "a guest" && i == 0) ? g.capitalize : g)}.to_sentence.html_safe %> 
-    left kudos on
+    <% givers_list = @kudo_givers["#{commentable.id}"] %>
```

Comment:
Is the change from @kudo_givers["#{commentable.class.name}_#{commentable.id}"] to @kudo_givers["#{commentable.id}"] necessary?

The loops I went though trying to get this to work the other way :(

And are you sure you want to generate an email with just the text "left kudos on Thing" if @kudo_givers["#{commentable.id}"] turns out to be nil? Shouldn't @kudo_givers be checked for nil before the email templates are used at all?

What happens if the work is deleted between the kudos being given and the resque job being processed. I think the right thing to do would be to exit out of the function and pretend the email had been delivered however.....

Also, the original code only capitalised the first name if it was the text "a guest", but now it will capitalise any user name at the start of the list. I don't think we want that - if the user's name is "e e cummings" then it shouldn't be changed to "E e cummings" (people care passionately about these things :).

ack, Will consider best way to do this.